core/src/main/resources/static/js/envs.js

```diff
@@ -705,6 +705,11 @@ app.controller("envsCtrl", function($scope, $http, $location, $window) {
 
         $scope.addNewEnv = function() {
 
+                var defRepFactor = parseInt($scope.addNewEnv.defrepfctr,10)
+                var maxRepFactor = parseInt($scope.addNewEnv.maxrepfctr,10)
+                var defPartitionFactor = parseInt($scope.addNewEnv.defparts,10)
```

Comment:
```suggestion
                var defPartitions = parseInt($scope.addNewEnv.defparts,10)
```

Reply:
well that was a silly mistake to make gimme two minutes and I'll fix it